data/team.yml

```diff
@@ -180,3 +180,16 @@ members:
 
 
       Aside from software, she enjoys recharging outdoors and discovering new hobbies, from cycling to dance classes.
+
+  - category: Engineering
+    image: "robert.jpg"
+    name: "Robert Czanik"
+    teaser: "Machine Learning Engineer | Full Stack Data Scientist | Braai master | Former astronomer "
+    description: >
+      Robert has a strong background in the hard sciences and is a seasoned data science professional. He is passionate about everything data related, from Postgres databasis, deep learning models to everything that build, deploys or shows the value in data.
```

Comment:
Did you actually mean 'databases', or is 'databasis' an actual thing? 😛

Reply:
Sticking to English, so it should be databases - and resolved.